input/examples/procedure-stentinsertion.xml

```diff
@@ -32,6 +32,14 @@
         </coding>
         <text value="Occlusion of right coronary artery"/>
     </reasonCode>
+    <bodySite>
```

Comment:
If this body site is added to this example you must update Procedure.code.text to include the body site information as per the guidance in AU core.

Reply:
Done